### ISSUE TITLE
"Dash" symbols inside code block

### ISSUE THREAD

**Issue by lampslave** [dcwatson/bbcode#27]
Dashes inside code block should remain unchanged, but in 1.0.26:
```
print(bbcode.render_html('[code]--[/code]'))
<code>&ndash;</code>

print(bbcode.render_html('[code]---[/code]'))
<code>&mdash;</code>
```

**Comment by dcwatson** [comment]
Thanks for the report, I just released 1.0.27 with the fix.

**Comment by lampslave** [comment]
Thank you!
